sass/components/_chart-controls.scss

```diff
@@ -212,3 +236,57 @@
         }
     }
 }
+.smartcharts-480 {
+    .cq-chart-controls {
+        .ciq-menu {
+            width: 47px;
+            .cq-menu-btn {
+                > .ic-icon{
+                    padding: .35em 0em;
+                }
+                .ic-subtitle{
+                    display: none;
+                }
+            }
+            .budget{
+                left: 28px;
```

Comment:
do you have to hard code the value of this class for each width configuration? Small changes would easily cause it to be misaligned. If it is positioned relative to the icon you may not need to do this.

Reply:
@brucebinary Becuase of menu structure I can not handle it by `relative position`. also as we have specific `width` for each size, I think it's ok.

Reply:
@shayanbinary hm... ok. I'll take a look at this someday.